app/src/main/res/values/styles.xml

```diff
@@ -337,35 +337,13 @@
     </style>
 
     <style name="SimpleDraweeViewPlaceholder">
```

Comment:
Should we rename the `SimpleDrawee` to another proper name since it has been removed?

Reply:
sure!